src/node_options.cc

```diff
@@ -546,6 +546,10 @@ EnvironmentOptionsParser::EnvironmentOptionsParser() {
             &EnvironmentOptions::prof_process);
   // Options after --prof-process are passed through to the prof processor.
   AddAlias("--prof-process", { "--prof-process", "--" });
+  AddOption("--experimental-import-map",
```

Comment:
```suggestion
  AddOption("--import-map",
```

Do we need `experimental` in the name?

Reply:
Hm, I mean I am happy to not, but I thought this was standard practice until things were stable.

Reply:
Absolutely, let’s not repeat what we’ve done for `--experimental-loader`, let’s keep things as clearly experimental until they’re ready

Reply:
Going to call this resolved for now. I think we should remove the prefix later once we get some real world experiments done.